crates/sui-graphql-rpc/tests/examples_validation_tests.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-#[cfg(feature = "pg_integration")]
+// #[cfg(feature = "pg_integration")]
```

Comment:
Not sure why is this disabled, we don't need this cfg feature anymore?

Reply:
ah ... good catch lol